src/index.ts

```diff
@@ -35,11 +35,41 @@ import parseRdpcProgramUpdateEvent from "eventParsers/parseRdpcProgramUpdateEven
   const expressApp = express();
   const statusReporter = applyStatusReport(expressApp)("/status");
   expressApp.use(
-    "/",
+    "/api-docs",
     swaggerUi.serve,
     swaggerUi.setup(yaml.load(path.join(__dirname, "./assets/swagger.yaml")))
   );
 
+  expressApp.post("/index/program/:program_id", async (req, res) => {
+    const programId = req.params.program_id;
+    try {
+      logger.info(
+        `received request to index program ${programId}, validating program id...`
+      );
+      // validate programId:
+      const regex = new RegExp(
+        "^[A-Z0-9][-_A-Z0-9]{2,7}[-](([A-Z][A-Z])|(\bINTL\b))$"
+      );
+      const found = programId.match(regex);
+
+      if (!found) {
+        return res
+          .status(404)
```

Comment:
`404` is `not found`, I think a `400` for `bad request` is probably more appropriate.

`404` implies that the app has gone looking for the resource but couldn't find it, which would be relevant if the user provided a valid `programId` that doesn't exist in the system. This isn't critical for us here but could be a thing to add later, but it'll require another dependency on program service so I'm not sure it's worth it since it's only us using this endpoint for convenience.